### ISSUE TITLE
Lovell: News Release Detail Lovell TRICARE pages erroring

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#20177]
## Description
All of the Lovell TRICARE News release detail pages are 404-ing. Some Examples:

### Example URLs: 
- https://main-kqjsor0i3mjdwsy9gojxhhyvdzh0wubb.tugboat.vfs.va.gov/lovell-federal-health-care-tricare/news-releases/lovell-fhcc-to-hold-next-patient-tele-town-hall-may-13/
- https://main-kqjsor0i3mjdwsy9gojxhhyvdzh0wubb.tugboat.vfs.va.gov/lovell-federal-health-care-tricare/news-releases/lovell-fhcc-meets-permanent-housing-placement-target-for-homeless-veterans-in-2022/
- https://main-kqjsor0i3mjdwsy9gojxhhyvdzh0wubb.tugboat.vfs.va.gov/lovell-federal-health-care-tricare/news-releases/lovell-federal-health-care-center-director-promotes-the-sergeant-first-class-heath-robinson-honoring-our/

### Additional info
Documentation from Lovell investigation: https://vfs.atlassian.net/wiki/spaces/AP1/pages/3978330239/Lovell+Investigation?force_transition=67d80d8a-5be8-4ebf-9a1f-8c8687f9afe5

### Acceptance Criteria

- [ ] Lovell TRICARE pages should be available on next-build
- [ ] UX review

### QA/Product testing and Approval

- [ ] The Lovell TRICARE detail page requirements:
   - [ ] Must display on the next-build tugboat instance at all screen sizes
   - [ ] `title` tag must say `{articleTitle} | Lovell Federal Health Care | Veterans Affairs' (casing might not be exact, there is a separate global issue for this; this also doesn't say TRICARE or VA and that's ok because it's consistent with Prod)
   - [ ] The URL must be `/lovell-federal-health-care-tricare/news-releases/`
   - [ ] The breadcrumbs must be (the casing of the crumbs may not match exactly, there is a separate global issue for this):
      - [ ] VA.gov home and direct to the main VA.gov homepage (the homepage will be a 404 on next-build)
      - [ ] Lovell Federal health care - TRICARE and direct to `/lovell-federal-health-care-tricare/` (this page will be a 404 on next-build)
      - [ ] News releases and direct to the TRICARE news listing page
      - [ ] Title of the article and be an anchor link to the main content section of the page
   - [ ] The switcher will probably be missing, there is a separate ticket for this
   - [ ] All article content and data displays (note: the print and download links will not use the proper component at this time, there are separate tickets to update these)
   - [ ] "See all news releases" link displays using the `<va-link>` component with active prop and directs to the TRICARE news listing page
   - [ ] Feedback button displays properly and opens the Medallia feedback form when activated
   - [ ] Last updated text does not display
- [ ] The Lovell VA detail page remains unchanged

**Comment by moanu404** [comment]
Interesting and weird - all these pages load up for me locally, just not on Tugboat perhaps? cc @laflannery 🤔 

<img width="1451" alt="Image" src="https://github.com/user-attachments/assets/d419cdeb-86d0-44ab-a3a6-ec9d3bf9670a" />

<img width="1409" alt="Image" src="https://github.com/user-attachments/assets/6df55249-79a0-4ae4-9c1f-e3fce84814f3" />

**Comment by laflannery** [comment]
@moanu404 Yes, generally any link in a ticket is going to be Tugboat because I don't use local.

- This is the main n-b tugboat: https://main-kqjsor0i3mjdwsy9gojxhhyvdzh0wubb.tugboat.vfs.va.gov/
- And the main TRICARE listing is a 404: https://main-kqjsor0i3mjdwsy9gojxhhyvdzh0wubb.tugboat.vfs.va.gov/lovell-federal-health-care-tricare/news-releases (which is [a separate ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/20142))
- And then the individual pages are also 404ing as described above: https://main-kqjsor0i3mjdwsy9gojxhhyvdzh0wubb.tugboat.vfs.va.gov/lovell-federal-health-care-tricare/news-releases/lovell-fhcc-to-hold-saturday-flu-shot-clinics-in-october-for-veterans-military-retirees-and-military-family/

**Comment by gracekretschmer-metrostar** [comment]
Will likely pick this up today.

**Comment by moanu404** [comment]
@gracekretschmer-metrostar I see these links seem to no longer 404, I suspect something may have fixed this along the way!
